tests/shared/src/test/scala/scala/meta/tests/parsers/BasePositionSuite.scala

```diff
@@ -9,6 +9,7 @@ import munit.TestOptions
 
 abstract class BasePositionSuite(defaultDialect: Dialect) extends ParseSuite {
   import scala.meta._
+  import scala.meta.contrib.TreeOps
```

Comment:
i don't think we use any `.contrib` stuff in the main codebase, i'd keep it that way.

Reply:
The only problem is that for Scala 3 we haven't yet implemented traversers so some other options are:
- add simple tree ops to scala.meta._
- have separate code for Scala 2 and 3 tests
- inline it for the tests

Reply:
could add a simple one for the tests. maybe one that uses `.andThen` and  `.applyOrElse` or `.lift` as superior to `.ifDefinedAt`.

Reply:
Inlined collect into the MoreHelpers object